packages/react/src/ActionMenu/ActionMenu.examples.stories.tsx

```diff
@@ -656,3 +656,102 @@ export const DynamicAnchorSides = () => {
     </>
   )
 }
+
+export const AnchorPositionScrollBug = () => {
```

Comment:
wondering if we should make this a dev story instead? is there a reason we want to make this bug highly visible?